bin/jest.js

```diff
@@ -224,7 +224,7 @@ function runJest() {
     return;
   }
 
-  const cwd = process.cwd();
+  const cwd = process.cwd().toLowerCase();
```

Comment:
Won't it break non-Windows users? As file paths are case sensitive on Linux, UNIXes.